crmd/crmd_lrm.h

```diff
@@ -148,6 +148,8 @@ int lrm_state_register_rsc(lrm_state_t * lrm_state,
                            const char *provider, const char *agent, enum lrmd_call_options options);
 int lrm_state_unregister_rsc(lrm_state_t * lrm_state,
                              const char *rsc_id, enum lrmd_call_options options);
+void lrm_state_foreach_remote(void lrm_state_func(lrm_state_t *lrm_state,
+                                                  gpointer user_data), gpointer user_data);
```

Comment:
For consistency with function arguments elsewhere, let's use the (*lrm_state_func) syntax here. That should probably be in the style guide ...